.envrc

```diff
@@ -3,3 +3,4 @@ eval "$(dev-env/bin/dade assist)"
 
 # Load private overrides
 [[ -f .envrc.private ]] && source_env .envrc.private
+
```

Comment:
Accidentally committed? it already had a trailing newline.

Reply:
Yeah, my first attempt at defining the variables in a central location was putting them in `.envrc` but it looked like `ci/build.yml` doesn't actually use that, so I moved it to `dev-env/lib/dade-dump-profile`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/9515/commits/3caa73317702ae2705f6110e9842deb2405d4b18